extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -136,6 +150,12 @@ export class AmpStoryPage extends AMP.BaseElement {
     /** @private {?Element} */
     this.playMessageEl_ = null;
 
+    /** @private {?Element} */
+    this.expandedViewOverlay_ = null;
+
+    /** @private {?Element} */
+    this.expandedComponent_ = null;
```

Comment:
Sorry, I think I'm confused: why does each page have an expanded component, but there's also a global expanded component in the store?

Reply:
It's similar concept to the `this.previousClickedEl_` in `amp-story-tooltip`.  We need to store the previously expanded element, because we set the global one in the store to `null` when closing the overlay. But we still need to know what element was previously expanded so we can reset its z-index.

I think I could move this logic to the tooltip, but I would still need to keep parts of it for the page.